go/libkb/globals.go

```diff
@@ -238,6 +238,7 @@ func (g *GlobalContext) PushShutdownHook(sh ShutdownHook) {
 }
 
 func (g *GlobalContext) ConfigureConfig() error {
+	RemoteSettingsRepairman(g)
```

Comment:
Repairman only proceeds if there is a `Roaming\Keybase\config.json` and no `Local\Keybase\config.json`. It will check this every time the process starts, until presumably we retire the repairman + force upgrade.

In testing it seems like the log files don't get copied over - maybe it is a timing thing? I assumed we could live with that, but if we treat it as a catastrophic failure, we'll (I'll) have to figure it out.